zulipterminal/helper.py

```diff
@@ -131,11 +129,12 @@ def set_count(id_list: List[int], controller: Any, new_count: int) -> None:
     all_msg = controller.view.home_button
     all_pm = controller.view.pm_button
     for id in id_list:
-        user_id = messages[id]['sender_id']
+        # FIXME
+        #user_id = messages[id]['sender_id']
```

Comment:
This needs to be addressed properly, since `sender_id` data is not available for streams. Should I do a lookup from `index['messages']` ?